recipe/build.sh

```diff
@@ -1,19 +1,30 @@
 #!/bin/bash
 
-set -euxo pipefail
+set -uxo pipefail
 
 if [[ "${target_platform}" == "linux-ppc64le" ]]; then
   export CFLAGS="${CFLAGS//-fno-plt/}"
   export CXXFLAGS="${CXXFLAGS//-fno-plt/}"
 fi
 
 if [[ "${CONDA_BUILD_CROSS_COMPILATION:-0}" == "1" ]]; then
-  CMAKE_ARGS="${CMAKE_ARGS} -DLLVM_TABLEGEN_EXE=$BUILD_PREFIX/bin/llvm-tblgen -DNATIVE_LLVM_DIR=$BUILD_PREFIX/lib/cmake/llvm"
+  # Can't have llvmdev in BUILD_PREFIX as llvmdev conflicts with the clang compiler
+  # TODO: remove conflict between llvmdev and clang compiler
+  conda create -p $BUILD_PREFIX/llvm llvmdev=$PKG_VERSION -c conda-forge --yes --quiet
```

Comment:
This seems like a superset of https://github.com/conda-forge/clangdev-feedstock/issues/196, where we can't mix our OSX compilers with newer libclang, or in this case: llvm.

Would this be solvable by using `clang_bootstrap` here? Otherwise I can't see how we can build clang against a shared libllvm (though currently we don't distinguish that precisely, we just have `llvmdev == {{ version }}` in host/run everywhere; so perhaps this can be broken down to be more finer-grained, or we build clang against a static llvm)